Makefile

```diff
@@ -94,22 +96,22 @@ modd-precheck:
 	fi; \
 
 watch: modd-precheck ## Run both server and webapp watching for changes
-	modd
+	env FOCALBOARD_BUILD_TAGS='$(BUILD_TAGS)' modd
 
 watch-single-user: modd-precheck ## Run both server and webapp in single user mode watching for changes
-	env FOCALBOARDSERVER_ARGS=--single-user modd
+	env FOCALBOARDSERVER_ARGS=--single-user FOCALBOARD_BUILD_TAGS='$(BUILD_TAGS)' modd
 
 watch-server-test: modd-precheck ## Run server tests watching for changes
-	modd -f modd-servertest.conf
+	env FOCALBOARD_BUILD_TAGS='$(BUILD_TAGS)' modd -f modd-servertest.conf
 
 server-test: ## Run server tests
-	cd server; go test -race -v -count=1 ./...
+	cd server; go test -tags='$(BUILD_TAGS)' -race -v -count=1 ./...
 
 webapp: ## Build webapp.
 	cd webapp; npm run pack
 
 watch-plugin: modd-precheck ## Run and upload the plugin to a development server
-	modd -f modd-watchplugin.conf
+	env FOCALBOARD_BUILD_TAGS='$(BUILD_TAGS)' modd -f modd-watchplugin.conf
```

Comment:
Does need the plugin the sqlite compilation flag? In theory, the plugin shouldn't use Sqlite ever.

Reply:
It's true that the plugin doesn't need it. I propagated it in case we'd like to use other `BUILD_TAGS`, but I'm happy to remove it if preferred